code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -1,399 +1,399 @@
-/mob/living/carbon/human/examine(mob/user)

```

Comment:
This is problem and not readable difference at all. I assume you need to merge master into your branch to see if it will fix it.